include/aikido/common/SplineFwd.hpp

```diff
@@ -0,0 +1,172 @@
+// This file is part of Eigen, a lightweight C++ template library
+// for linear algebra.
+//
+// Copyright (C) 20010-2011 Hauke Heibel <hauke.heibel@gmail.com>
```

Comment:
Are we going to update the head description?

Reply:
I think we should keep this license as we copied this code from Eigen, and will add our license on top of this license for the part we modify/add.